app/backend/src/couchers/jobs/handlers.py

```diff
@@ -1055,3 +1058,61 @@ def gen_randomized_coords(user_id, lat, lng):
 
 update_randomized_locations.PAYLOAD = empty_pb2.Empty
 update_randomized_locations.SCHEDULE = timedelta(hours=1)
+
+
+def send_event_reminders(payload: empty_pb2.Empty):
+    """
+    Sends reminders for events that are 24 hours away to users who marked themselves as attending.
+    """
+    logger.info("Sending event reminder emails")
+
+    with session_scope() as session:
+        now_time = now()
+        lower_bound = now_time + timedelta(hours=23.5)
+        upper_bound = now_time + timedelta(hours=24.5)
```

Comment:
The background job is not guaranteed to run exactly every 60 minutes (and one cannot in general guarantee that either), so this should use some kind of mechanism to guarantee idempotency.

I suppose the two options are: add a flag to the event, or add a flag to the user occurrence. The flag means keeping track of if we've sent a reminder yet.